web/Table/Table.html

```diff
@@ -0,0 +1,304 @@
+      <th scope="col">Name<button class="btn Hide btn-link"/><button class="btn Devide btn-link"/></th>
+      <th scope="col">Student ID<button class="btn Hide btn-link"/><button class="btn Devide btn-link"/></th>
+      <th scope="col">Email<button class="btn Hide btn-link"/><button class="btn Devide btn-link"/></th>
+      <th scope="col">Paper #1</th>
+      <th scope="col">Paper #2</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th scope="row">1</th>
+      <td> 
+      <button class="btn fa btn-link"/>
+      <button class="btn edit btn-link"/>
+<button class="btn cancel btn-link"/>
+<button class="btn send btn-link"/>
+      
+      </td>
+      <td>Giacomo</td>
+      <td>Guilizzoni</td>
+      <td>21GD01</td>
+      <td>peldi@inbox.lv</td>
+      <td>A Collaborative Code Review Pl...</td>
+      <td>A comparative study of hybrid...</td>
+    </tr>
+    <tr>
+      <th scope="row">2</th>
+      <td> 
+      <button class="btn fa btn-link"/>
+      <button class="btn edit btn-link"/>
+<button class="btn cancel btn-link"/>
+<button class="btn send btn-link"/>
+      
+      </td>
+      <td>Marco</td>
+      <td>Botton</td>
+      <td>22GD02</td>
+      <td>peldi@gmail</td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <th scope="row">3</th>
+      <td>
+       <button class="btn fa btn-link"/>
+      <button class="btn edit btn-link"/>
+<button class="btn cancel btn-link"/>
+<button class="btn send btn-link"/>
+      </td>
+      <td>Mariah</td>
+      <td>Maclachlan</td>
+      <td>22GD03</td>
+      <td>oreon@mail.com</td>
+      <td>A comparison of scrum and Knab...</td>
+      <td></td>
+    </tr>
+    <tr>
+      <th scope="row">4</th>
+      <td>
+       <button class="btn fa btn-link"/>
+      <button class="btn edit btn-link"/>
+<button class="btn cancel btn-link"/>
+<button class="btn send btn-link"/>
+      </td>
+      <td>Liberty</td>
+      <td>Valerie</td>
+      <td>23GD04</td>
+      <td>itcode@hotmail.com</td>
+      <td>A comparative Study of UML To...</td>
+      <td>A Comparison of Metrics for...</td>
+    </tr>
+    <tr>
+      <th scope="row">5</th>
+      <td>
+          <button class="btn fa btn-link"/>
+      <button class="btn edit btn-link"/>
+<button class="btn cancel btn-link"/>
+<button class="btn send btn-link"/>
+      </td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <th scope="row">6</th>
+     <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+     <td></td>
+     <td></td>
+     <td></td>
+     <td></td>
+     <td></td>
+     <td></td>
+     <td></td>
+     <td></td>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+   
+  </tbody>
+</table>
+
+</div>
+</html>
```

Comment:
HTML file should be in the `views` folder.

Reply:
Not resolved, but we need the code to be able to complete other tasks, so merging

Reply:
Not resolved, but we need the code to be able to complete other tasks, so merging